features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/conf/site.json

```diff
@@ -15,7 +15,8 @@
         "responseSigningEnabled":"true",
         "keyStoreName" :"",
         "passive" : "true",
-        "signRequests" : "true"
+        "signRequests" : "true",
+        //"acsURL" : "https://localhost:9443/store/jagg/jaggery_acs.jag" //In passive or request signing mode, use only if app generated Assertion Consumer Service URL needs to be overidden
```

Comment:
Please note this is loaded as a .js file so the comments are working. Here we are using comments so that it is easier for user to do configurations by un-commenting.